docs/CONFIG-PROPERTIES.md

```diff
@@ -51,6 +51,7 @@
 | netdump.topic.postonboard.interval | integer in seconds | 1 day | how frequently (in seconds) can be netdumps of the same topic published after device has been onboarded |
 | netdump.topic.maxcount | integer | 10 | maximum number of netdumps that can be published for each topic. The oldest netdump is unpublished should a new netdump exceed the limit.
 | netdump.downloader.with.pcap | boolean | false | include packet captures inside netdumps for download requests. However, even if enabled, TCP segments carrying non-empty payload (i.e. content which is being downloaded) are excluded and the overall PCAP size is limited to 64MB. |
+| disable.switch.ni.arpsnoop | boolean | false | disable ARP Snooping on switch Network Instance, may need device reboot to take effect |
```

Comment:
I noticed that most boolean config options vaguely follow template: `<topic>.<item>.enable.<feature>`
I would therefore propose something like `network.switch.enable.arpsnoop`
WDYT?

Reply:
Sure @milan-zededa

Reply:
yes.